presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java

```diff
@@ -101,7 +100,7 @@
 public class TestPrestoS3FileSystem
 {
     private static final int HTTP_RANGE_NOT_SATISFIABLE = 416;
-
+    private static final String S3_DIRECTORY_OBJECT_CONTENT_TYPE = "application/x-directory; charset=UTF-8";
```

Comment:
It doesn't look like this ever gets used.  Also, it would be great if we could have tests that check that it works both with and without the charset specified.

Reply:
This is used in testEmptyDirectory(). Will add test for without the charset in the string.

Reply:
@rschlussel : I have added the test for content type without charset too. Please have a look. Thanks!